storage/migrations/16_validator_staking_history.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+CREATE TABLE chain.validator_balance_history
+(
+    id base64_ed25519_pubkey NOT NULL,
+    epoch UINT63 NOT NULL,
+    PRIMARY KEY (id, epoch),
+    escrow_balance_active UINT_NUMERIC NOT NULL,
+    escrow_balance_debonding UINT_NUMERIC NOT NULL
```

Comment:
I think it would be useful to also have the number of delegators here.

Then we can use data from this table also on the "dashboard" page to display the current number of all delegators.

Reply:
added!